utils/contexts/auth/AuthProvider.tsx

```diff
@@ -49,7 +49,9 @@ export const AuthProvider = ({
       // signout firebase
       firebaseAuth.signOut();
       // destroy cookie auth data
-      destroyCookie(undefined, COOKIE_USER_AUTH);
+      destroyCookie(undefined, COOKIE_USER_AUTH, {
```

Comment:
Well, turn out that `destroyCookie` method is searching for a cookie that matches the name in the same path let's say `/` then delete it. But if we destroy the same named cookie but different path, it will just add a new one with empty value in it. So yeah 3rd party problem, glad we adapt! Blimey!

Reply:
Well. Glad we tackle this one already. Great job! Merging soon!